WebApp/src/components/BasicTest/BasicTestContainer.tsx

```diff
@@ -0,0 +1,30 @@
+import './BasicTestContainer.css';
+import { IonText, IonButton } from '@ionic/react';
+import React from 'react';
+
+interface ContainerProps {}
+
+const ExploreContainer: React.FC<ContainerProps> = () => {
+  const quotes = [
+    'Georges is the best programmer',
+    'Etien is the best programmer',
+  ];
+
+  const [quote, setQuote] = React.useState(quotes[0]);
+
+  return (
+    <div className="container">
+      <IonText color="primary">{quote}</IonText>
+      <div />
+      <IonButton
+        color="secondary"
+        className="quoteButton"
+        onClick={() => setQuote(quotes.find((x) => x !== quote)!)}
```

Comment:
Pour un set c'est discutable mais il y a une façon de penser qu'on ne devrais pas mettre de fonctions directement dans le render et de les déclarer plus haut. Personnellement si c'est juste une ligne ça m'importe peu.
`const handleClick = () => {};`
`const handleClick = useCallback(() => {}, []);`

Reply:
![image](https://user-images.githubusercontent.com/26386997/119726401-11748c80-be3f-11eb-8565-8cffe01a6da9.png)

Je suis down qu'on ne mette pas directement les fonctions dans le render. À He-Who-Must-Not-Be-Named on fait ça et le code est rendu dégueulasse, aucune distinction entre la view et les fonctionnalités du composants pi c'est vraiment pas débugable